pkg/scorecard.go

```diff
@@ -168,21 +169,23 @@ func runScorecard(ctx context.Context,
 	go runEnabledChecks(ctx, repo, request, checksToRun, resultsCh)
 
 	if os.Getenv(options.EnvVarScorecardExperimental) == "1" {
-		// Get configuration
-		rc, err := repoClient.GetFileReader("scorecard.yml")
-		// If configuration file exists, continue. Otherwise, ignore
-		if err == nil {
-			defer rc.Close()
+		r := findConfigFile(repoClient)
+
+		logger := sclog.NewLogger(sclog.DefaultLevel)
+
+		if r != nil {
+			defer r.Close()
 			checks := []string{}
 			for check := range checksToRun {
 				checks = append(checks, check)
 			}
-			c, err := config.Parse(rc, checks)
+			c, err := config.Parse(r, checks)
 			if err != nil {
-				logger := sclog.NewLogger(sclog.DefaultLevel)
 				logger.Error(err, "parsing configuration file")
 			}
 			ret.Config = c
+		} else {
+			logger.Info(fmt.Sprintf("no maintainer annotation: %v", err))
```

Comment:
I lean towards the opposite. Logging if a maintainer annotation file is found and used, not if it's absent.

For example, golangci-lint spits this out, and we could do the same:
```
INFO [config_reader] Used config file .golangci.yml
```

Reply:
Plus, I don't think `err` is set appropriately there and shouldn't be in the message.

Reply:
Fixed, only logging if annotation can't be parsed.